src/DynamoCore/Engine/ShortestQualifiedNameReplacer.cs

```diff
@@ -90,7 +91,40 @@ public override AssociativeNode VisitIdentifierListNode(IdentifierListNode node)
                 RightNode = rightNode,
                 Optr = Operator.dot
             };
-            return leftNode != newLeftNode ? node : RewriteNodeWithShortName(node);
+            return node;
+        }
+
+        public override AssociativeNode VisitIdentifierNode(IdentifierNode node)
```

Comment:
I know many public methods in this visiter are missing summaries - /// - does it make sense to add them? Or atleast add them to new visitors?

Reply:
I think it is implicit in the name of the method/design pattern. If any, I think the class should have some docs if it doesn't already.

Reply:
okay, we don' t need to address this now, but we should investigate silencing the xml doc warnings these methods throw during build then.